src/matrix/wishart.jl

```diff
@@ -179,7 +238,7 @@ end
 
 function _rand_params(::Type{Wishart}, elty, n::Int, p::Int)
     n == p || throw(ArgumentError("dims must be equal for Wishart"))
-    ν = elty( n + 1 + abs(10randn()) )
+    ν = elty( n - 1 + abs(10randn()) )
```

Comment:
Can you explain?

Reply:
`_rand_params ` should return random nonsingular parameters. You just need `ν > n - 1` for that.